tox.ini

```diff
@@ -41,3 +41,9 @@ commands =
 deps = coverage
 skip_install = true
 commands = coverage erase
+
+[testenv:docs]
+description = invoke sphinx-build to build the HTML docs
+basepython = python3.7
+deps = sphinx >= 1.7.5, < 2
```

Comment:
By running this as a tox command we can let tox install sphinx in a Python 3 environment and skip those steps as part of docs generation